source/vibe/inet/webform.d

```diff
@@ -259,7 +259,20 @@ private bool parseMultipartFormPart(InputStream stream, ref FormFields form, ref
 		auto data = cast(string)stream.readUntil(boundary);
 		form.addField(name, data);
 	}
-	return stream.readLine(max_line_length) != "--";
+	
+	enforce(stream.leastSize >= 2);
+	auto ub = stream.peek();
```

Comment:
This is not a good idea in general (leastSize could return just 1 even if the connection is still open and could yield more data, and `peek` is not required to return anything at all). But in this case it's possible to just use `ubyte[2] ub; stream.read(ub);` instead. It has to be either "--" or "\r\n". Both of them terminate the form part.